### ISSUE TITLE
Verification sets and test sets differ greatly in accuracy

### ISSUE THREAD

**Issue by SXQ-STUDY** [XuJiacong/PIDNet#50]
When training with training set, the accuracy of verification set can reach 78.5, while when training from scratch with training set and verification set, the accuracy can only reach 77.2 on the test server. What is the reason

**Comment by tcwang23** [comment]
您好！
请问您的问题解决了吗？
我也遇到了同样的问题，PIDNet在cityscapes数据集val可达到78.2的精度，但在test只有75.5的miou。
请问是如何解决的训练验证相差过大的问题？期待您的回复，谢谢。
